Traceback (most recent call last):
File "/Users/matthew/local/opt/google_appengine/google/appengine/tools/adaptive_thread_pool.py", line 150, in WorkOnItems
  status, instruction = item.PerformWork(self.__thread_pool)
File "/Users/matthew/local/opt/google_appengine/google/appengine/tools/bulkloader.py", line 675, in PerformWork
  transfer_time = self._TransferItem(thread_pool)
File "/Users/matthew/local/opt/google_appengine/google/appengine/tools/bulkloader.py", line 1054, in _TransferItem
  download_result = self.request_manager.GetEntities(self)
File "/Users/matthew/local/opt/google_appengine/google/appengine/tools/bulkloader.py", line 1274, in GetEntities
  query = key_range_item.key_range.make_directed_datastore_query(self.kind)
File "/Users/matthew/local/opt/google_appengine/google/appengine/ext/key_range/__init__.py", line 246, in make_directed_datastore_query
  query = self.filter_datastore_query(query)
File "/Users/matthew/local/opt/google_appengine/google/appengine/ext/key_range/__init__.py", line 175, in filter_datastore_query
  return EmptyDatastoreQuery(query.kind)
AttributeError: 'Query' object has no attribute 'kind'
[INFO    ] An error occurred. Shutting down...
.....[ERROR   ] Error in Thread-7: 'Query' object has no attribute 'kind'

[INFO    ] Have 83 entities, 0 previously transferred
[INFO    ] 83 entities (0 bytes) transferred in 2.5 seconds

